server/controllers/authentication/login.js

```diff
@@ -1,3 +1,30 @@
-module.exports = (req, res) => {
-  res.send('loging route');
+const joi = require('joi');
+const { sign } = require('jsonwebtoken');
+const bcrypt = require('bcryptjs');
+const { checkUsername } = require('../../database/queries/authentication/checkData');
+const { loginSchema } = require('../../helpers/validation-schema');
+
+module.exports = (req, res, next) => {
+  const memberInfo = { ...req.body };
+  const { error } = joi.validate(memberInfo, loginSchema);
+  if (!error) {
+    checkUsername(memberInfo.username)
+      .then(({ rows: member }) => {
```

Comment:
@Fatmasiam I have a very minor suggestion: If you are sick of typing `member[0]` you can use brackets to destructure more in this line, like this: `.then(({ rows: [ member ] }) => {`
Then below, you can just write (for example) `member.password`